### PR TITLE
add structure to install section; add docker-compose instructions

### PR DESCRIPTION
**Pull request by GeorgLink**
I think we should separate out the three ways to install GrimoireLab and have separate pages for each but have one central page for the projects.json and setup.cfg.

This pr adds pages for Docker, Docker Compose, and PIP. 

This pr adds content for the main install page and the Docker Compose install method.

PIP and Docker need to be done later.

### CODE REVIEW

File: docs/install/docker-compose.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+* Pass: admin
+
+## Configure GrimoireLab
+
+Configuration is done through two files:
+
+* projects.json
+* setup.cfg
+
+Docker-Compoes is by default configured to look for these files in the 'default-grimoirelab-settings' folder.
```
Comment (valeriocos) [inline docs/install/docker-compose.md]: small typo -> compoes

File: docs/install/docker-compose.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+```
+
+## Error Handling
+
+If you experience an error, run `docker-compose` up without the `-d` flag. That will output all messages and give insight to problems. 
+
+
+### Unable to connect to Elasticsearch on first start?
+
+By default, Elasticsearch will not work. This is a known problem: you need to [set `vm.max_map_count' to at least `262144'](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#_set_vm_max_map_count_to_at_least_262144). See the [elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#_set_vm_max_map_count_to_at_least_262144) for how to do it on Linux, Mac, or Windows.
```
Comment (valeriocos) [inline docs/install/docker-compose.md:87]: Not sure we have support for windows

File: docs/install/index.md [hunk 3]
```diff
@@ -8,13 +8,35 @@ has_toc: false
...
\ No newline at end of file
+## Define analysis scope
+
+### Tell GrimoireLab what projects to collect data from
+
+GrimoireLab needs to know what projects to collect data from. 
+The default [projects.json](https://github.com/chaoss/grimoirelab/blob/master/default-grimoirelab-settings/projects.json) file is configured to collect metrics about the GrimoireLab git repository. 
+
+To start collecting data about the projects that matter to you, you need to define projects and data endpoints in the project.json file.
+The [GrimoireLab tutorial section on the projects file](https://chaoss.github.io/grimoirelab-tutorial/sirmordred/projects.html) explains how the file needs to be built.
```
Comment (valeriocos) [inline docs/install/index.md]: As general rule (when possible), it's probably better to point to the doc of the corresponding GitHub repo, in this case: https://github.com/chaoss/grimoirelab-sirmordred#projectsjson

**Review by valeriocos — COMMENTED** [review]
Thank @GeorgLink for the PR, overall it looks OK, I left some comments.

File: docs/install/docker-compose.md (hunk 2 above)
Comment (GeorgLink) [reply docs/install/docker-compose.md:87]: Does it matter with Docker containers?

File: docs/install/index.md (hunk 3 above)
Comment (GeorgLink) [reply docs/install/index.md]: Good point! I'll make the change.

**Comment by GeorgLink** [comment]
@valeriocos thanks for the review. I made the changes.

On the Windows support; The instructions I'm referring to are for elasticsearch containers and those are also available on Windows.

File: docs/install/docker-compose.md (hunk 2 above)
Comment (valeriocos) [reply docs/install/docker-compose.md:87]: I don't think so, but I ignore the steps to set the vm_max_map_count in windows

**Review by valeriocos — APPROVED** [review]
Thank you @GeorgLink , LGTM

**Pull request merged**